src/Sections/EntitiesSection/Entities/Leader.ts

```diff
@@ -0,0 +1,52 @@
+import Entity, { CommonEntityOptions } from '../Entity';
+import { BoundingBox, boundingBox_t } from 'Internals/BoundingBox';
+import { Dxfier } from 'Internals/Dxfier';
+import { vec2_t, point3d } from 'Internals/Helpers';
+
+export enum ArrowHeadFlags {
+    Disabed = 0,
+    Enabled = 1,
+}
+
+export enum PathType {
+    StraightLine = 0,
+    Spline = 1,
+}
+
+export interface LeaderVertex {
+    point: vec2_t;
+}
+
+export interface LeaderOptions extends CommonEntityOptions {
+    flags?: ArrowHeadFlags;
```

Comment:
its only a flag not flags, I just looked in the spec they call it **Arrowhead flag**